Upstream-Status: Inappropriate [distribution]

Take out lscpu stuff from the code

2011/03/17
Saul Wold <saul.wold@intel.com>
Nitin A Kamble <nitin.a.kamble@intel.com>

Index: util-linux-2.20.1/sys-utils/Makefile.am
===================================================================
--- util-linux-2.20.1.orig/sys-utils/Makefile.am
+++ util-linux-2.20.1/sys-utils/Makefile.am
@@ -17,12 +17,6 @@ usrsbin_exec_PROGRAMS += ldattach tunelp
 dist_man_MANS += dmesg.1 ctrlaltdel.8 cytune.8 setarch.8 \
 		ldattach.8 tunelp.8 rtcwake.8 fsfreeze.8 fstrim.8
 
-if HAVE_CPU_SET_T
-usrbin_exec_PROGRAMS += lscpu
-lscpu_SOURCES = lscpu.c $(top_srcdir)/lib/cpuset.c $(top_srcdir)/lib/strutils.c
-dist_man_MANS += lscpu.1
-endif
-
 endif
 
 cytune_SOURCES = cytune.c cyclades.h
